pinot-common/src/test/java/org/apache/pinot/common/utils/LoggerUtilsTest.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.utils;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class LoggerUtilsTest {
+  private static final Logger LOGGER = LoggerFactory.getLogger(LoggerUtilsTest.class);
+
+  @Test
+  public void testGetAllLoggers() {
+    List<String> allLoggers = LoggerUtils.getAllLoggers();
+    Assert.assertEquals(allLoggers.size(), 1);
+    Assert.assertEquals(allLoggers.get(0), "root");
+  }
+
+  @Test
+  public void testGetLoggerInfo() {
+    Map<String, Object> rootLoggerInfo = LoggerUtils.getLoggerInfo("root");
+    Assert.assertEquals(rootLoggerInfo.get("name"), "root");
+    Assert.assertEquals(rootLoggerInfo.get("level"), "WARN");
+    Assert.assertNull(rootLoggerInfo.get("filter"));
+
+    Assert.assertNull(LoggerUtils.getLoggerInfo("notExistLogger"));
+  }
+
+  @Test
+  public void testChangeLoggerLevel() {
+    Assert.assertEquals(LoggerUtils.getLoggerInfo("root").get("level"), "WARN");
+    for (String level : ImmutableList.of("WARN", "INFO", "DEBUG", "ERROR", "WARN")) {
+      LoggerUtils.setLoggerLevel("root", level);
+      System.out.println("Current logger level is " + level);
+      loggerAllLevels();
```

Comment:
We should verify whether the log has the certain level enabled, e.g. `assertTrue(LOGGER.isWarnEnabled())`